boards/arm/circuitdojo_feather_nrf9160/Kconfig.board

```diff
@@ -0,0 +1,15 @@
+# Circuit Dojo nRF9160 Feather configuration
+
+# Copyright (c) 2018-2020 Nordic Semiconductor ASA
+# Copyright (c) 2020 Circuit Dojo LLC
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF9160_SICA
+
+config BOARD_CIRCUITDOJO_FEATHER_NRF9160
+	bool "Circuit Dojo nRF9160 Feather"
+
+config BOARD_CIRCUITDOJO_FEATHER_NRF9160_NS
```

Comment:
Can you remove the underscore from "_NS" here?

Reply:
...and in similar occurrences, if applicable

Reply:
Can do!